src/js/index.js

```diff
@@ -14,21 +14,23 @@ model.init();
 var Sidebars = require('./components');
 
 var g = model.Scene.child('marks.group'),
+    g2 = g.child('marks.group'),
+    g3 = g2.child('marks.group'),
     p = model.pipeline('cars'),
     p2 = model.pipeline('jobs'),
     p3 = model.pipeline('gapminder');
 
 // Pre-populate state with one rect, one symbol, one text & one line mark
-g.child('marks.rect');
+g3.child('marks.rect');
 g.child('marks.symbol');
-g.child('marks.line');
+g2.child('marks.line');
 g.child('marks.text');
 g.child('marks.area');
 
 Promise.all([
-  p._source.init({url: 'http://vega.github.io/vega-editor/app/data/cars.json'}),
-  p2._source.init({url: 'http://vega.github.io/vega-editor/app/data/jobs.json'}),
-  p3._source.init({url: 'http://vega.github.io/vega-editor/app/data/gapminder.json'})
+  p._source.init({url: '/data/cars.json'}),
```

Comment:
(and to clarify, I _will_ rebase that out, but I'm interested in adding this in anyway at least for a time because it dramatically speeds up the speed of reload on the app)